packages/@react-aria/select/src/HiddenSelect.tsx

```diff
@@ -92,4 +92,6 @@ export function HiddenSelect<T>(props: HiddenSelectProps<T>) {
         value={state.selectedKey} />
     );
   }
+
+  return null;
```

Comment:
what was going on here?

Reply:
react throws an error if you return undefined